internal/cri/server/status.go

```diff
@@ -94,5 +95,39 @@ func (c *criService) Status(ctx context.Context, r *runtime.StatusRequest) (*run
 		}
 		resp.Info["lastCNILoadStatus"] = defaultStatus
 	}
+
+	intro, err := c.client.IntrospectionService().Server(ctx, &ptypes.Empty{})
+	if err != nil {
+		return resp, err
+	}
+	cond := &runtime.RuntimeCondition{
+		Type:   ContainerdHasNoDeprecationWarning,
+		Status: true,
+	}
+	if len(intro.Deprecations) > 0 {
+		cond.Status = false
+		reasons := make([]string, len(intro.Deprecations))
+		messages := make(map[string]string, len(intro.Deprecations))
+		for i, d := range intro.Deprecations {
+			reasons[i] = d.ID
```

Comment:
@SergeyKanzhelev In the `/etc/containerd/config.toml` do you want to see an allow list (or deny list?) for deprecation IDs?

Reply:
I think a deny list (anything not on the list will be exposed) makes more sense here, because a system (GKE, in our case) is introducing a deprecated usage explicitly, and therefore the system should hide it explicitly as well. 
In general i think any deprecation should be exposed by default unless explicitly configured otherwise.

Reply:
if this can be even more scoped - like excluding a very specific mirrors set - it will be even better. Not sure how common this ask will be beyond GKE, but I assume it is not unique for GKE

Reply:
If we do end up adding a deny list, I think it should only impact the propagation via CRI rather than the introspection service itself.  `ctr deprecations list` should show everything regardless of who was responsible for causing the use of deprecated behavior.

Reply:
Added `IgnoreDeprecationWarnings []string` to the CRI config